model/config.go

```diff
@@ -248,26 +250,26 @@ var ServerTLSSupportedCiphers = map[string]uint16{
 
 type ServiceSettings struct {
 	SiteURL                                           *string  `access:"environment,authentication,write_restrictable"`
```

Comment:
This one should be provided because is used [here](https://github.com/mattermost/mattermost-webapp/blob/master/components/admin_console/admin_definition.jsx#L3494-L3499) and if not present fails